engine/core/IgeEntity.js

```diff
@@ -2560,7 +2560,7 @@ var IgeEntity = IgeObject.extend({
 					ige.pixi.viewport.removePlugin('follow');
 					this.emit('stop-follow');
 				}
-				var texture = ige.entityTrack.trackEntityById[entityId]._pixiTexture || ige.entityTrack.trackEntityById[entityId]._pixiText || ige.entityTrack.trackEntityById[entityId];
+				var texture = ige.entityTrack.trackEntityById[entityId]._pixiTexture || ige.entityTrack.trackEntityById[entityId]._pixiText; // removing this || ige.entityTrack.trackEntityById[entityId];
```

Comment:
without this deletion, we were recursively adding ourselves to the call stack essentially by calling `this.destroy()`